src/components/pageTemplates/NodeArticlePage.tsx

```diff
@@ -10,7 +10,9 @@ interface NodeArticlePageProps {
 }
 
 export function NodeArticlePage({ node, ...props }: NodeArticlePageProps): JSX.Element {
-  const { title, field_lead, created, field_content, langcode} = node
+  const { title, field_lead, created, field_content, langcode, published_at } = node;
+  const articleDate = published_at !== null ? published_at : created;
```

Comment:
Scheduled publishing actually seems to use `created` and not `published_at`. Can you double check if this is correct?